.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,153 @@
+name: Ansible CI
```

Comment:
rename the file to ci.yaml. Also is this file coming from `avi-dev`?

Reply:
Done